examples/sensors-viewer/src/components/SensorViewer.jsx

```diff
@@ -0,0 +1,221 @@
+            loading,
+            fetched,
+        } = this.state;
+        return (
+            <Container className="px-0 py-4" fluid>
+                <Container>
+                    <Row>
+                        <Col xs={12} className="p-0">
+                            <h5 className="sensor-main-div text-center text-uppercase
+                font-weight-bold text-white bg-success mb-5 px-0 py-3">
+                                Sensor Viewer
+                            </h5>
+                            <Col xs={12} className="sensor-id-search-div px-5 pt-5 pb-4">
+                                <Row className="main-row col-sm-7 align-items-center mx-auto">
+                                    <Col sm={2} xs={12} className={"p-0"}>
+                                        <span className="font-weight-normal">Device ID:</span>
+                                    </Col>
+                                    <Col sm={10} xs={12}>
+                                        <InputGroup>
+                                            <FormControl
+                                                placeholder="Enter ID Here"
+                                                aria-label="Enter ID Here"
+                                                aria-describedby="basic"
+                                                className="bg-white font-weight-normal rounded"
+                                                name="deviceID"
+                                                onChange={this.handleChange}
+                                            />
+                                            <InputGroup.Append className="ml-3">
+                                                <Button onClick={this.handleSubmit} disabled={loading}
+                                                        className="bg-success border-success
+                                text-uppercase font-weight-normal px-4
+                                text-decoration-none rounded">
+                                                    {loading ? <Spinner
+                                                        as="span"
+                                                        animation="border"
+                                                        size="sm"
+                                                        role="status"
+                                                        aria-hidden="true"
+                                                    /> : "Submit"}
+                                                </Button>
+                                            </InputGroup.Append>
+                                        </InputGroup>
+                                    </Col>
+                                </Row>
+                                <Row className="card-main-row mt-5">
+                                    <Col xs={12} className="device-status-div px-3 pb-4">
+                                        {fetched &&
```

Comment:
the `a && b` and `a || b` when handling data that is not boolean is considered a "hack" or "dirty JS trick". If ths is expected to return a value, you can use the ternary operator `a ? b : ""` or build your object in a constant before the `return` statement and add it to the template later on